class-pinterest-for-woocommerce.php

```diff
@@ -794,6 +849,25 @@ public static function is_connected() {
 		}
 
 
+		/**
+		 * Checks if connected and on a Business account.
+		 *
+		 * @return boolean
+		 */
+		public static function is_business_connected() {
+
+			if ( ! self::is_connected() ) {
+				return false;
+			}
+
+			$account_data = self::get_setting( 'account_data' );
+			$is_partner   = isset( $account_data['is_partner'] ) ? (bool) $account_data['is_partner'] : false;
+
+			return $is_partner;
```

Comment:
We don't need the `$is_partner` variable here, and we can also simplify the conditional a bit.

```suggestion

			return (bool) ( $account_data['is_partner'] ?? false );
```